orderbook/src/api.rs

```diff
@@ -1,3 +1,4 @@
+mod common;
```

Comment:
We have put other common code directly into `api.rs` so I think `convert_get_orders_error_to_reply` can also go there.

Reply:
Cool, didn't know this. Will do.

Reply:
Done!